README.md

```diff
@@ -37,21 +37,25 @@ cd openscap
 
 2) To build the library you will also need the following build dependencies
 (some of these are optional, if they are not detected, openscap will be compiled
-without respective optional features). On RHEL 7 / Fedora, the command to
```

Comment:
Why was the note about RHEL7 / Fedora removed? Doesn't seem to be related to python2.

Reply:
I thought that as there is only Python2 on RHEL7, I will take the mention about it out. Fedora also installs Python 2 by default.

Reply:
But the rest of the packages are RHEL / Fedora specific. There is a separate section for ubuntu. It doesn't read to me like the note about RHEL 7 and Fedora is there just because of python2.

Reply:
OK, I have put back the sentence with `RHEL / Fedora / CentOS` list.